CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ include(${Slicer_USE_FILE})
 add_subdirectory(BreachWarningLight)
 add_subdirectory(Viewpoint)
 add_subdirectory(LumpNav)
+add_subdirectory(TipToSurfaceDistance)

```

Comment:
Thanks for the contribution TipToSurfaceDistance is a useful IGT feature in general, so it should go into the SlicerIGT extension. However, we already have a very similar module there: BreachWarning.

BreachWarning module determines the closest point on a model from tooltip, computes the distance, makes it available to other modules through MRML, changes color of a model and/or gives sound signal if the tip gets too close (inside) the model, supports any number of nodes, runs in the background (no need to activate the GUI at all), usable from both Python and C++ (as it's a C++ loadable modeul), etc. BreachWarning is already used by LumpNav.

Drawing crosshair and line trajectory are nice features that could be added to the existing BreachWarning module and then we would not have the huge redundancy between TipToSurfaceDistance and BreachWarning. Do you think you could implement these features in BreachWarning?